diff --git a/dist/client/index.d.ts b/dist/client/index.d.ts
index 97c05f00f219ee40cae115a1964399aee1fd3e9f..7e39af43f5d117b2c2d7fe4d6d4423ba98e208b5 100644
--- a/dist/client/index.d.ts
+++ b/dist/client/index.d.ts
@@ -15,16 +15,16 @@ export type ClientApi = ApiFromModules<{
     client: ReturnType<R2["clientApi"]>;
 }>["client"];
 type RunQueryCtx = {
-    runQuery: GenericQueryCtx<GenericDataModel>["runQuery"];
+    runQuery: GenericActionCtx<GenericDataModel>["runQuery"];
 };
 type RunMutationCtx = {
-    runQuery: GenericQueryCtx<GenericDataModel>["runQuery"];
-    runMutation: GenericMutationCtx<GenericDataModel>["runMutation"];
+    runQuery: GenericActionCtx<GenericDataModel>["runQuery"];
+    runMutation: GenericActionCtx<GenericDataModel>["runMutation"];
 };
 type RunActionCtx = {
     runAction: GenericActionCtx<GenericDataModel>["runAction"];
-    runQuery: GenericQueryCtx<GenericDataModel>["runQuery"];
-    runMutation: GenericMutationCtx<GenericDataModel>["runMutation"];
+    runQuery: GenericActionCtx<GenericDataModel>["runQuery"];
+    runMutation: GenericActionCtx<GenericDataModel>["runMutation"];
 };
 export declare class R2 {
     component: ComponentApi;
